src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkContentTests/MsftToSbSdkFiles.diff

```diff
@@ -3,8 +3,10 @@ index ------------
 --- a/msftSdkFiles.txt
 +++ b/sbSdkFiles.txt
 @@ ------------ @@
+ ./host/fxr/
  ./host/fxr/x.y.z/
  ./host/fxr/x.y.z/libhostfxr.so
++./library-packs/
```

Comment:
Will update the exclusions list to account for this one. It should've been included in the last PR update since it went along with those changes.